README.md

```diff
@@ -1,17 +1,52 @@
-# eZ Dataflow
+# Code-Rhapsodie EzDataflow Bundle
 
-The eZ Dataflow bundle is intent to manage external data sources import into your EZPlatform instance.
+EzDataflowBundle is a bundle to integrate the Code-Rhapsodie DataflowBundle into eZ Platfom 2.0+.
+These dataflows can be piloted from an UI integrated in eZ Platform Admin UI.
```

Comment:
Dataflows from DataflowBundle are managed with an UI integrated in eZ Platform Admin UI.